### PR TITLE
Add ability to answer a specific renderer

### PR DESCRIPTION
**Pull request by karaggeorge**
Wasn't sure if creating a new method or overloading the existing one would be best. I've gone with overloading in this PR. Can you take a look and if this works, I can update docs and typings. 

If not, I can move it to a new method, which will look very similar, and it might be called something like `ipcMain.answerSpecificRenderer` or something like that

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -56,30 +56,59 @@ export interface MainProcessIpc extends IpcMain {
...
+    ```
+    import {ipcMain as ipc} from 'electron-better-ipc';
+
+    ipc.answerRenderer('get-emoji', async emojiName => {
+      const emoji = await getEmoji(emojiName);
+      return emoji;
+    });
+    ```
+    */
+    <DataType, ReturnType = unknown>(
```
Comment (sindresorhus) [inline index.d.ts:77]: Do you really have to nest these inside `answerRenderer` to do overloads?

File: index.d.ts [hunk 2]
```diff
@@ -56,30 +56,59 @@ export interface MainProcessIpc extends IpcMain {
...
-	```
-	*/
-	answerRenderer<DataType, ReturnType = unknown>(
-		channel: string,
-		callback: (
-			data: DataType,
-			browserWindow: BrowserView
-		) => ReturnType | PromiseLike<ReturnType>
-	): () => void;
+  answerRenderer: {
```
Comment (sindresorhus) [inline index.d.ts:59]: Use tab-indentation. Applies to the whole file here.

**Comment by karaggeorge** [comment]
Fixed the comments 😄 
Just a note, I wasn't able to run the spectron tests locally, and I don't think they are run on CI. I think I got the strings correctly but I'm not 100% sure.

**Comment by sindresorhus** [comment]
It would be great if you could ensure the tests are still passing. Just upgrade the `electron` and `spectron` dependencies to latest and then run `npm run test-with-spectron`.

**Comment by sindresorhus** [comment]
Thanks! 👌

**Pull request merged**